internal/providers/msentraid/msentraid.go

```diff
@@ -39,6 +48,21 @@ func (p Provider) AuthOptions() []oauth2.AuthCodeOption {
 	return []oauth2.AuthCodeOption{}
 }
 
+func (p Provider) CheckTokenScopes(token *oauth2.Token) {
```

Comment:
We added tests in a pairing session. How do you usually handle reviews of changes done while pairing? Do you consider it reviewed because two pair of eyes were on it, or should we ask a third person (i.e. @denisonbarbosa)  to review it?

Reply:
Normally, we consider pairing as "someone else looked at it and it’s a review". I will just review the additional test commits and mark it explicitely as reviewed here.